src/page_components/listing/ListingDetails.scss

```diff
@@ -0,0 +1,11 @@
+.details {
+  --desktop-width: 66.666667%;
```

Comment:
You could change this to `--desktop-width: calc(100% * (2/3))`. isn't really any more concise though